authorization/storage_authorization_test.go

```diff
@@ -77,6 +109,115 @@ func TestAuth(t *testing.T) {
 				}
 			},
 		},
+		{
+			name:  "read",
+			setup: setup,
+			results: func(t *testing.T, store *authorization.Store, tx kv.Tx) {
+				/*auths, err := store.ListAuthorizations(context.Background(), tx, influxdb.AuthorizationFilter{})
+				if err != nil {
+					t.Fatal(err)
+				}
+
+				if len(auths) != 10 {
+					t.Fatalf("expected 10 authorizations, got: %d", len(auths))
+				}
+
+				expected := []*influxdb.Authorization{}
+				for i := 1; i <= 10; i++ {
+					expected = append(expected, &influxdb.Authorization{
+						ID:     influxdb.ID(i),
+						Token:  fmt.Sprintf("randomtoken%d", i),
+						OrgID:  influxdb.ID(i),
+						UserID: influxdb.ID(i),
+						Status: "active",
+					})
+				}
+				if !reflect.DeepEqual(auths, expected) {
+					t.Fatalf("expected identical authorizations: \n%+v\n%+v", auths, expected)
+				}*/
+
+				expectedAuth := influxdb.Authorization{
+					ID:     influxdb.ID(1),
+					Token:  fmt.Sprintf("randomtoken%d", 1),
+					OrgID:  influxdb.ID(1),
+					UserID: influxdb.ID(1),
+				}
+
+				authByID, err := store.GetAuthorizationByID(context.Background(), tx, influxdb.ID(1))
+				if err != nil {
+					t.Fatal("Unexpectedly could not acquire Authorization by ID")
+				}
+
+				if !reflect.DeepEqual(authByID, expectedAuth) {
+					t.Fatalf("expected identical authorizations: \n%+v\n%+v", authByID, expectedAuth)
+				}
+
+				authByToken, err := store.GetAuthorizationByToken(context.Background(), tx, fmt.Sprintf("randomtoken%d", 1))
+				if err != nil {
+					t.Fatal("cannot get authorization by Token")
+				}
+
+				if !reflect.DeepEqual(authByToken, expectedAuth) {
+					t.Fatalf("expected identical authorizations: \n%+v\n%+v", authByToken, expectedAuth)
+				}
+
+			},
+		},
+		{
+			name:  "update",
+			setup: setupForUpdateTest,
+			results: func(t *testing.T, store *authorization.Store, tx kv.Tx) {
+				/*auths, err := store.ListAuthorizations(context.Background(), tx, influxdb.AuthorizationFilter{})
```

Comment:
same here

Reply:
Yeah I'm getting the rebase sorted out @jsteenb2 Thanks!

Reply:
heads up, likely easier to just cherry pick these commits out, squash them down and pick em out 🤷

Reply:
Yeah that sounds like a good idea. I ended up instead updating my local branch to the latest copy and creating new PR.

Reply:
Looks like that took care of my Jenkins issues

Reply:
oh sweet, nicely done 👌